nodestream_akamai/appsec_coverage/appsec-coverage.yaml

```diff
@@ -0,0 +1,26 @@
+- implementation: nodestream-akamai.appsec_coverage:AkamaiAppSecCoverageExtractor
```

Comment:
There are a bunch of places with `nodestream-akamai`. I think these should all be `nodestream_akamai`.